javascript/javascript3/week2/main.js

```diff
@@ -0,0 +1,151 @@
+// Movies exercise
+// This exercise is repetition of array functions. you dont have to use chaining or anything fancy. Just fetch the movies and use the correct array function to get the following movies:
+
+// Fetch movies from this api:
+
+// Create an array of bad movies
+// Creat an array of bad movies since year 2000
+
+let result = fetch(
+  "https://gist.githubusercontent.com/pankaj28843/08f397fcea7c760a99206bcb0ae8d0a4/raw/02d8bc9ec9a73e463b13c44df77a87255def5ab9/movies.json"
+).then((response) =>
+  response
+    .json()
+    .then((arrOfAllmovies) => {
+      const badMovie = 4;
+      const arrayOfBadMovies = arrOfAllmovies.filter(
+        (movie) => movie.rating < badMovie
+      );
+      console.log(arrayOfBadMovies);
+      return arrayOfBadMovies;
+    })
+    .then((arrayOfBadMovies) => {
+      const yearLimit = 2000;
+      const arrayOfBadMoviesSince2000 = arrayOfBadMovies.filter(
+        (movie) => movie.year > yearLimit
+      );
+      console.log(arrayOfBadMoviesSince2000);
+    })
+);
```

Comment:
const moviesJsonUrl =
  "https://gist.githubusercontent.com/pankaj28843/08f397fcea7c760a99206bcb0ae8d0a4/raw/02d8bc9ec9a73e463b13c44df77a87255def5ab9/movies.json";
const allMovies = fetch(moviesJsonUrl)
.then((response) =>response.json())
.then((result) => {
   const arrayOfBadMovies = result.filter ((movie) => {
       return movie.rating < 4  })
       console.log(arrayOfBadMovies)

       const BadMoviesSince2000 = arrayOfBadMovies.filter (movie => {
        return  movie.year > 2000
    })
    console.log(BadMoviesSince2000)
    }
    
    );
you take a look how you can solve like this exercise :-)

Reply:
Super.